`make dashboard` in demo project errors out for me on Ubuntu

I'm just trying out the demo project and I was able to run `make init`, install the CLI, start a cluster, and go to the dashboard and it worked.  However, after I shut it down and ran it again, (`make down; make up`) I tried the command it gave me to open the dashboard `make dashboard` and got this error:

```
$ make dashboard
Couldn't get a file descriptor referring to the console
Makefile:77: recipe for target 'dashboard' failed
make: *** [dashboard] Error 1
```

I tried make reset, which seemed to work fine, but I got the same error when running `make dashboard`.  I can still access the dashboard, just have to be sure to store the login token somewhere, since I can't access it via make anymore.

--- a/demo/Makefile
+++ b/demo/Makefile
@@ -74,7 +74,7 @@ token:
 
 # Open dashboard in browser
 dashboard:
-	@open http://dashboard.${CLUSTER_DOMAIN}?token=${$(MAKE) token}
+	@URL=http://dashboard.${CLUSTER_DOMAIN}?token=${$(MAKE) token} && xdg-open $$URL || open $$URL
 
 # Run a command in the VM via SSH
 ssh-cmd:

--- a/demo/Makefile
+++ b/demo/Makefile
@@ -74,7 +74,8 @@ token:
 
 # Open dashboard in browser
 dashboard:
-	@URL=http://dashboard.${CLUSTER_DOMAIN}?token=${$(MAKE) token} && xdg-open $$URL || open $$URL
+	@CMD=open && which xdg-open > /dev/null && CMD=xdg-open; \
+	$$CMD http://dashboard.${CLUSTER_DOMAIN}?token=${$(MAKE) token}
 
 # Run a command in the VM via SSH
 ssh-cmd:

